command/output.go

```diff
@@ -68,6 +73,20 @@ func (c *OutputCommand) Run(args []string) int {
 		return 1
 	}
 
+        if allOutputs {
+                ks := make([]string, 0, len(mod.Outputs))
+                for k, _ := range mod.Outputs {
+                        ks = append(ks, k)
+                }
+                sort.Strings(ks)
+
+                for _, k := range ks {
+                        v := mod.Outputs[k]
+                        c.Ui.Output(fmt.Sprintf("%s = %s", k, v))
```

Comment:
I was more thinking of HCL-compatible output, e.g. `variable_name = "space here"` or `another_variable = "dash-here"`, but then I found it's actually ["broken" elsewhere](https://github.com/hashicorp/terraform/blob/21b0a03/terraform/state.go#L614-627) too, so I think it's ok to keep it as is in this PR.

The escaping can be fixed all together in different PR.